templates/azure/fips.html

```diff
@@ -0,0 +1,405 @@
+     <p><a href="https://www.brighttalk.com/webcast/6793/452063" class="p-button--positive"><span class="p-link--external">Watch the webinar</span></a></p>
+    </div>
+    <div class="col-6">
+      <div class="jsBrightTALKEmbedWrapper" style="width:100%; height:100%; position:relative;background: #ffffff;"><script class="jsBrightTALKEmbedConfig" type="application/json">{ "channelId" : 6793, "language": "en-US", "commId" : 452063, "displayMode" : "standalone", "height" : "auto" }</script><script src="https://www.brighttalk.com/clients/js/player-embed/player-embed.js" class="jsBrightTALKEmbed"></script></div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-12">
+      <h2>FIPS certified Ubuntu components</h2>
+      <table>
+        <thead>
+          <tr>
+            <th style="width: 10%;">Ubuntu LTS</th>
+            <th style="width: 20%;">FIPS certified component</th>
+            <th>Description</th>
+            <th style="width: 10%;">Version</th>
+            <th style="width: 15%;">CMVP Certificate</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>18.04</td>
+            <td>Azure Kernel</td>
+            <td>Kernel optimised for use in Azure clouds</td>
+            <td>4.15</td>
+            <td>3683</td>
+          </tr>
+          <tr>
+            <td>18.04</td>
+            <td>OpenSSL</td>
+            <td>General purpose cryptographic library that includes TLS implementation</td>
+            <td>1.1.1</td>
+            <td>3622</td>
+          </tr>
+          <tr>
+            <td>18.04</td>
+            <td>OpenSSH client</td>
+            <td>SSH server application for operating systems</td>
+            <td>7.9p1</td>
+            <td>3633</td>
+          </tr>
+          <tr>
+            <td>18.04</td>
+            <td>OpenSSL server</td>
+            <td>SSH client application for operating systems</td>
+            <td>7.9p1</td>
+            <td>3632</td>
+          </tr>
+          <tr>
+            <td>18.04</td>
+            <td>StrongSWAN</td>
+            <td>IPSec based VPN solution library</td>
+            <td>5.6.2</td>
+            <td>3648</td>
+          </tr>
+        </tbody>
+      </table>
+      <br><br>
+      <table>
+        <thead>
+          <tr>
+            <th style="width: 10%;">Ubuntu LTS</th>
+            <th style="width: 20%;">FIPS certified component</th>
+            <th>Description</th>
+            <th style="width: 10%;">Version</th>
+            <th style="width: 15%;">CMVP Certificate</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>16.04</td>
+            <td>Linux kernel (generic)</td>
+            <td>The Linux kernel cryptographic library</td>
+            <td>4.4.0.1002</td>
+            <td>2962</td>
+          </tr>
+          <tr>
+            <td>18.04</td>
```

Comment:
double check this column, this one and below should be `16.04`. Also the tables don't work too well on mobile view.

![image](https://user-images.githubusercontent.com/58959073/100756457-279c1580-33e5-11eb-8a06-bf6971dcbbf3.png)

 I had this issue recently but there is a pattern on vanilla that creates cards on small screens- see [here](https://vanillaframework.io/docs/base/tables#responsive). I used the pattern on https://ubuntu.com/kubernetes/compare

Reply:
Fixed version. 

Great suggestion with the responsive tables. Done.